app/src/main/java/org/feature/fox/coffee_counter/data/local/database/dao/UserDao.kt

```diff
@@ -21,6 +24,9 @@ interface UserDao {
     @Update
     suspend fun insertPurchase(purchase: Purchase)
 
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertImage(image: Image)
+
```

Comment:
Pls add Db at the end of the method name to stay consistent and provide better Overview